source/release-notes.txt

```diff
@@ -10,6 +10,34 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.36.4
+----------------
+
+*Released April 27, 2023*
+
+New Features:
+
+- Update add data icon to plus with circle from download (:issue:`COMPASS-6494`)
+- Show import progress in toast, make import background 
+  (:issue:`COMPASS-6540`, :issue:`COMPASS-6555`)
+- Import progress (:issue:`COMPASS-6721`)
+- Update new connection text to new window (:issue:`COMPASS-6723`)
+
+Bug Fixes:
+
+- Remove re-count when not available (:issue:`COMPASS-5179`, :issue:`COMPASS-6649`)
+- Fill autocomplete on tab (:issue:`COMPASS-6695`)
+- Show error border when focused (:issue:`COMPASS-6724`)
+- Compass readonly allows to drop namespaces from the sidebar (:issue:`COMPASS-6687`)
+- Fixes the problem of refresh button on collection tab not refreshing 
+  the collection stats (:issue:`COMPASS-6738`)
+- Update windows config file fetching location one folder up (:issue:`COMPASS-6527`)
+- If listCSVFields() or analyzeCSVFields() fails it will display the error 
+  in the modal (:issue:`COMPASS-6737`)
+
```

Comment:
Past release notes sometimes skip the link, sometimes use this format, and sometimes say, "Full Changelog available on Github" 

Consider using the named link instead of the comparison.

Reply:
Fixed